.github/workflows/continuous-delivery.yaml

```diff
@@ -31,16 +30,27 @@ jobs:
         run: npm ci --ignore-scripts --no-audit --no-progress --prefer-offline
       - name: Build
         run: npm run build
-      - id: release
+      - env:
+          GITHUB_TOKEN: ${{ secrets.DOTTBOTT_TOKEN }}
+        id: release
         name: Release
         uses: ridedott/release-me-action@master
         with:
           commit-assets: |
             ./dist
           node-module: true
+      - if: steps.release.outputs.released == 'true'
+        name: Authenticate
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 1
+          persist-credentials: true
+          token: ${{ secrets.DOTTBOTT_TOKEN }}
       - if: steps.release.outputs.released == 'true'
         name: Tag
         run: |
+          git config --global user.name '{{ secrets.RIDEDOTT_GITHUB_USER_NAME }}'
```

Comment:
Maybe call the secret `DOTTBOTT_USER_NAME` and `DOTTBOTT_USER_EMAIL` ?

Dont see why RideDott and Github should be included in the secret's name.

Reply:
Resolved! I also added these secrets at an organization level, it seemed fitting.